modules/protocol/resources/META-INF/smithy/smithy4s.meta.smithy

```diff
@@ -11,6 +11,9 @@ metadata suppressions = [
 
 namespace smithy4s.meta
 
+@trait(selector: ":is(service, operation)")
```

Comment:
**question:** the doc page said it's for operations only (line 46 of `04-code-size.md` at this moment). Should this be updated?

Reply:
Yes